simple_history/tests/tests/test_models.py

```diff
@@ -677,19 +679,68 @@ def test_create_history_model_with_multiple_one_to_ones(self):
                 "exception."
             )
 
+    def verify_custom_model_name_feature(
+        self, model, expected_class_name, expected_table_name
+    ):
+        history_model = model.history.model
+        self.assertEqual(history_model.__name__, expected_class_name)
+        self.assertEqual(history_model._meta.db_table, expected_table_name)
+
     def test_instantiate_history_model_with_custom_model_name(self):
```

Comment:
Let's split this up into a bunch of different tests? I try to avoid a bunch of tests in the same function because it becomes very difficult to pinpoint bugs when something breaks

Reply:
will break up